build-system/tasks/visual-diff.rb

```diff
@@ -365,7 +377,7 @@ def snapshot_webpages(page, webpages, config)
         forbidden_css,
         loading_incomplete_css,
         loading_complete_css)
-    Percy::Capybara.snapshot(page, name: name)
+    Percy::Capybara.snapshot(page, name: name, enable_javascript: true)
```

Comment:
The other lines in here are sensible as standalone changes. This line here is a separate issue, and should be PR'ed and merged separately

Reply:
@danielrozenberg The other lines in this PR won't take effect without this one, since the problem we're trying to solve (https://github.com/percy/percy-capybara/issues/51) is that the images should not be double height when JS is enabled.

In any case, this PR isn't ready to merge because we have 8 new visual diffs due to double height images. @timhaines is investigating, so let's wait for an update from him. Meanwhile, this PR can remain un-merged.

Reply:
@danielrozenberg I've updated the scope of this PR based on the viewport discussion we just had.

Reply:
In that case, definitely split off the JS behaviour on Percy :)

Reply:
Done.